File: src/main/java/com/multicampus/gamesungcoding/a11ymarketserver/common/config/S3Config.java
package com.multicampus.gamesungcoding.a11ymarketserver.common.config;

import com.multicampus.gamesungcoding.a11ymarketserver.common.properties.S3StorageProperties;
import lombok.RequiredArgsConstructor;
import org.springframework.context.annotation.Configuration;

@Configuration
@RequiredArgsConstructor
public class S3Config {
    
    private final S3StorageProperties s3Properties;

}

File: src/main/java/com/multicampus/gamesungcoding/a11ymarketserver/common/config/S3Config.kt
package com.multicampus.gamesungcoding.a11ymarketserver.common.config

import com.multicampus.gamesungcoding.a11ymarketserver.common.properties.S3StorageProperties
import org.springframework.context.annotation.Configuration

@Configuration
class S3Config(
    private val s3Properties: S3StorageProperties
)